server/storage/s3.go

```diff
@@ -177,12 +190,3 @@ func (s *S3Storage) Put(ctx context.Context, token string, filename string, read
 }
 
 func (s *S3Storage) IsRangeSupported() bool { return true }
-
-func getAwsSession(accessKey, secretKey, region, endpoint string, forcePathStyle bool) *session.Session {
```

Comment:
let's keep a separated `getAwsConfig` function, to use at https://github.com/dutchcoders/transfer.sh/pull/559/files#diff-66e5af6447b0120f4ac7cd603a574ed6ca422560f3d03e8012f7a4f6acfb0326R32-R51